ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -25,7 +25,7 @@
     private static final By ADMIT_TO_INPATIENT = By.id("referenceapplication.realTime.simpleAdmission");
     private static final By EXIT_FROM_INPATIENT = By.id("referenceapplication.realTime.simpleDischarge");
     private static final By ACTIONS_DROPDOWN = By.cssSelector("#content span.dropdown-name");
-    private static final By MERGE_VISITS = By.cssSelector("#content div.actions.dropdown ul li:nth-child(2) > a");
+    private static final By MERGE_VISITS = By.cssSelector("#content div.actions.dropdown ul li:nth-child(3) > a");
```

Comment:
if this element moved as a result of revising the page, you can always expect such future revisions which would give us a maintanance burden, can you figure out a more unique way to reference this element